catalog/task/create-internal-request/0.6/README.md

```diff
@@ -0,0 +1,37 @@
+# create-internal-request
+
+Creates an InternalRequest resource to call IIB service
+
+## Parameters
+
+| Name | Description | Optional | Default value |
+|------|-------------|----------|---------------|
+| pipelineRunName | The name of the Parent PipelineRun for this task | No | `ir-$(context.pipelineRun.name)` |
+| request | Internal pipeline request name | No | |
+| params | Internal Request parameters | No | |
+| inputDataFile | Optional file to read data from | Yes | "" |
+| updateGenericResult | Should the task update the genericResult result  | Yes | "false" |
+| requestUpdateTimeout | Max seconds to wait until the status is updated | Yes | 360 |
+
+## Changelog
+
+### changes since 0.5
+- adds `updateGenericResult` parameter
```

Comment:
`adds UpdateGenericResult parameter to control whether the genericResult result is updated`

Reply:
fixed